### PR TITLE
use default ndk on macOS

### PR DESCRIPTION
**Pull request by ravinderjangra**
fixes: #37

### CODE REVIEW

**Comment by hannesa2** [comment]
This is the root cause https://github.com/ravinderjangra/android-ndk-toolchain-setup/blob/8cbc7305ba08258a142a7b66d2b516f716dfabb5/dist/index.js#L999

**Comment by hannesa2** [comment]
image

🥇

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -8,14 +8,21 @@ on: # rebuild any PRs and main branch changes
 
 jobs:
   build: # make sure build/ci work properly
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```
Comment (hannesa2) [inline .github/workflows/test.yml:11]: perfect

File: dist/index.js [hunk 2]
```diff
@@ -972,15 +972,27 @@ var __importStar = (this && this.__importStar) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 const core = __importStar(__webpack_require__(470));
 const exec = __importStar(__webpack_require__(986));
+const os = __importStar(__webpack_require__(87));
 class Ndk {
-    constructor(path) {
-        this.path = path;
+    constructor() {
```
Comment (hannesa2) [inline dist/index.js:977]: Because this file is auto-generated, can you delete this file from git and add it to `.gitignore` ?
This would avoid confusions

**Comment by hannesa2** [comment]
When you are done, please make a new tag.
I'm looking forward to test it in my projects

File: dist/index.js (hunk 2 above)
Comment (ravinderjangra) [reply dist/index.js:977]: we can remove the `index.js` because the GHA isn't compiling the code it's directly using it from the `dist` directory.

**Comment by hannesa2** [comment]
Please don't forget to make a new tag and push it

**Pull request merged**